tracer/samples/RabbitMQ.DistributedTracing/Receive/Receive.cs

```diff
@@ -94,6 +64,21 @@ public static void Main()
                     Console.ReadLine();
                 }
             }
+
+
+        }
+
+        static IEnumerable<string> GetHeaderValues(IDictionary<string, object> headers, string name)
+        {
+            if (headers == null)
+                return Enumerable.Empty<string>();
+
```

Comment:
In your sample, `headers` is never null, so I would probably just remove this check for simplicity

```suggestion
```

Reply:
I added it in case they don't set automatic instrumentation right, as it would break. But in between, I've moved this check upper in the chain. So indeed, can be removed